### PR TITLE
Rename to tasks

### PR DESCRIPTION
**Pull request by unkrich**
Renames the jobs section to tasks in anticipation of jobs where the nomenclature is

Task: Unit of work
Job: Encapsulation of some amount of work to be scheduled

### CODE REVIEW

File: monosi/tasks/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```
Comment (unkrich) [inline monosi/tasks/__init__.py:1]: This is so that we can use tasks across repositories. Disregard

File: monosi/tasks/base.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    @abc.abstractmethod
+    def run(self, *args, **kwargs):
+        raise NotImplementedError('Implementation for task does not exist.')
+
+class ProjectTask(TaskBase):
+    def __init__(self, args, config):
+        super().__init__(args, config)
+        self.project: Optional[Project] = None
+        self.task_queue: List[TaskBase] = []
```
Comment (iporollo) [inline monosi/tasks/base.py:35]: I wonder if there's a python type specifically for queue

**Review by iporollo — APPROVED** [review]
lgtm

File: monosi/tasks/base.py (hunk 2 above)
Comment (unkrich) [reply monosi/tasks/base.py:35]: Potentially - queue is misleading naming, this really is just a list cause order is arbitrary here any way technically.. at least right now

**Pull request merged**